Kernel/Arch/x86_64/Processor.cpp

```diff
@@ -1579,6 +1579,7 @@ void Processor::assume_context(Thread& thread, FlatPtr flags)
     // and then the scheduler lock
     VERIFY(Processor::in_critical() == 2);
 
+    u32 flags = 2 | (new_interrupts_state == InterruptsState::Enabled ? 0x200 : 0);
```

Comment:
Do we always need to set the second bit (2), by OR'ing it? This is not the same behaviour as when the code was located in execve.cpp.

Reply:
Yes, bit 2 is reserved and must be set to 1. (That's what we do every-where else where we set the flags)
The fact that we can have this kind of issue in the first place probably means we should add constexpr named values for these at some point, instead of hardcoding these everywhere